src/acceptor.rs

```diff
@@ -365,6 +379,14 @@ impl rustls_accepted {
     /// out_conn: An output parameter. The pointed-to pointer will be set
     ///   to a new rustls_connection only when the function returns
     ///   RUSTLS_RESULT_OK.
+    /// out_alert: An output parameter. The pointed-to pointer will be set
+    ///   to a new rustls_accepted_alert when, and only when, the function returns
+    ///   a non-OK result. The memory is owned by the caller and must eventually
+    ///   be freed with rustls_accepted_alert_free. The caller should call
+    ///   rustls_accepted_alert_write_tls to write the alert bytes to
+    ///   the TLS connection before freeing the rustls_accepted_alert.
+    ///
+    /// Only one of out_conn or out_alert will be set.
```

Comment:
This should also be updated to say "at most one."

Reply:
Fixed.